tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```diff
@@ -44,6 +41,7 @@ def __init__(
         enable_alltoall: bool,
         use_deepseek_fp8_block_scale: bool,
         use_w4a8_group_scaling: bool,
+        min_latency_mode: bool,
```

Comment:
Why do you need to pass the argument here? Should `self.min_latency_mode` just be false?